src/platform/pico/uart_channel.c

```diff
@@ -0,0 +1,240 @@
+#include "reactor-uc/platform/pico/uart_channel.h"
+#include "reactor-uc/platform/pico/pico.h"
+#include "reactor-uc/logging.h"
+#include "reactor-uc/serialization.h"
+
+#define UART_CHANNEL_ERR(fmt, ...) LF_ERR(NET, "UartPolledChannel: " fmt, ##__VA_ARGS__)
+#define UART_CHANNEL_WARN(fmt, ...) LF_WARN(NET, "UartPolledChannel: " fmt, ##__VA_ARGS__)
+#define UART_CHANNEL_INFO(fmt, ...) LF_INFO(NET, "UartPolledChannel: " fmt, ##__VA_ARGS__)
+#define UART_CHANNEL_DEBUG(fmt, ...) LF_DEBUG(NET, "UartPolledChannel: " fmt, ##__VA_ARGS__)
+
+extern Environment *_lf_environment;
```

Comment:
I believe you will get this declaration by doing `#include "reactor-uc/environment.h"`